### ISSUE TITLE
Items appear 2D in hand 

### ISSUE THREAD

**Issue by lilPNUTTY** [hatninja/RetroItems#3]
First issue is that items follow the camera even upwards which isn't bad but isn't accurate to how fast dropped items used to look. If this is intentional could you add another option to disable the items from facing upward? 
![image](https://user-images.githubusercontent.com/73256738/117559098-8038a400-b050-11eb-99d7-8b84b93be207.png)

Title says it all the items in regular gameplay go one pixel backward giving them a bit a depth and when the fast items option is enabled they look completely flat 
![image](https://user-images.githubusercontent.com/73256738/117559062-14563b80-b050-11eb-8394-1085c8684f65.png)

**Comment by hatninja** [comment]
Fair enough, i can try pulling off this effect and make an updated release later.
